qdrant-landing/content/articles/dedicated-service.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Vector Search as a dedicated service"
+short_description: "Some arguments on why vector search requires to be a dedicated service"
+description: "Some arguments on why vector search requires a dedicated service"
+social_preview_image: /articles_data/dedicated-service/social-preview.png
+small_preview_image: /articles_data/dedicated-service/preview/icon.svg
+preview_dir: /articles_data/dedicated-service/preview
+weight: -70
```

Comment:
the weight needs to be adjusted as a new article got published in the meantime 
https://qdrant.tech/articles/